### PR TITLE
CNTC and LSTMFCN models.

### PR DESCRIPTION
**Pull request by jnrusson1**
#### Reference Issues/PRs
Adds two new models #49 and #31 

#### What does this implement/fix? Explain your changes.

Adds a CNTC and LSTMFCN model to the package.

#### Does your contribution introduce a new dependency? If yes, which one? 

Yes the keras-self-attention package is included with the CNTC model. This may cause an issue so could be temporarily removed.

*Have commented out import and self attention layer for now.

### CODE REVIEW

**Comment by jnrusson1** [comment]
Current test build fails as it is using version 0.4.1 of sktime rather than 0.6.1

**Comment by ABostrom** [comment]
looks good jack. Need to figure out how we can fix the build failing using 0.41

File: setup.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def find_install_requires():
     """
 
     install_requires = [
-        'sktime==0.4.1',
+        'sktime==0.6.1',
```
Comment (ABostrom) [inline setup.py:53]: need to figure out how to get this to work. The original update to 0.61 should have fixed this.

Comment (TonyBagnall) [reply setup.py:53]: there is a requirements.txt file, I think I have  changed it on the refactor branch

Comment (jnrusson1) [reply setup.py:53]: I don't think that it will be possible to make the package work with 0.4.1 now as there have been a lot of name changes.

**Pull request merged**